frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -152,6 +152,20 @@ function FileManagerMenu:setUpdateItemTable()
                 },
             }
         }
+    elseif Device.isKindle() then
```

Comment:
Does the folder screensaver functionality not work on Kindle? Maybe I'm missing something, but couldn't you just change `if Device.isKobo() then` to `if Device.isKobo() or Device.isKindle() then` and add an extra menu item there? In fact Kobo also needs a way to disable screensaver globally besides just for a specific book.